src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -444,24 +479,203 @@ public void declaredLostEventIsPublished() {
+      .forLoanId(testLoanId)
+      .withServicePointId(servicePointId)
+      .on(declareLostDate)
+      .withNoComment();
+
+    declareLostFixtures.declareItemLost(builder);
+
+    JsonObject declareLostLoan = loansClient.getById(testLoanId).getJson();
+    JsonObject declareLostItem = itemsFixture.getById(itemId).getJson();
+ 
+    assertThat(declareLostItem, isDeclaredLost());
+    
+    Double finalAmountRemaining = declareLostLoan.getJsonObject("feesAndFines").getDouble("amountRemainingToPay");
+    assertEquals(finalAmountRemaining, 20.0, 0.01);
+
+    List<JsonObject> accounts = getAccountsForLoan(testLoanId);
+
+    assertThat(accounts, hasSize(2));
+    assertThat(getOpenAccounts(accounts), hasSize(1));
+  }
+  
+  @Test
+  public void shouldClearExistingFeesAndCloseLoanAsLostAndPaidIfLostandPaidItemDeclaredLostAndPolicySetNotToChargeFees() {
+	    final double lostItemProcessingFee = 20.0;
+	    UUID servicePointId = servicePointsFixture.cd1().getId();
+
+	    final LostItemFeePolicyBuilder lostPolicyBuilder = lostItemFeePoliciesFixture.ageToLostAfterOneMinutePolicy()
+	        .withName("age to lost with processing fees")
+	        .billPatronImmediatelyWhenAgedToLost()
+	        .withNoFeeRefundInterval()
+	        .withNoChargeAmountItem()
+	        .doNotChargeProcessingFeeWhenDeclaredLost()
+	        .chargeProcessingFeeWhenAgedToLost(lostItemProcessingFee);
+
+	    useLostItemPolicy(lostItemFeePoliciesFixture.create(lostPolicyBuilder).getId());
+
+	    AgeToLostResult agedToLostResult = ageToLostFixture.createLoanAgeToLostAndChargeFees(lostPolicyBuilder);
+	    UUID testLoanId = agedToLostResult.getLoanId();
+	    UUID itemId = agedToLostResult.getItemId();
+	    
+	    JsonObject AgeToLostItem = itemsFixture.getById(itemId).getJson();
+
+	    assertThat(AgeToLostItem, isAgedToLost());
+
+	    JsonObject itemFee = getAccountForLoan(testLoanId, "Lost item processing fee");
+	    
+	    assertThat(itemFee, hasJsonPath("amount", lostItemProcessingFee));
+
+	    final DateTime declareLostDate = now(UTC).plusWeeks(1);
+	    mockClockManagerToReturnFixedDateTime(declareLostDate);
+
+	    final DeclareItemLostRequestBuilder builder = new DeclareItemLostRequestBuilder()
+	      .forLoanId(testLoanId)
+	      .withServicePointId(servicePointId)
+	      .on(declareLostDate)
+	      .withNoComment();
+
+	    declareLostFixtures.declareItemLost(builder);
+
+	    JsonObject declareLostLoan = loansClient.getById(testLoanId).getJson();
+	    JsonObject declareLostItem = itemsFixture.getById(itemId).getJson();
+	 
+	    assertThat(declareLostItem, isLostAndPaid());
+	    
+	    Double finalAmountRemaining = declareLostLoan.getJsonObject("feesAndFines").getDouble("amountRemainingToPay");
+	    assertEquals(finalAmountRemaining, 0.0, 0.01);
+
+	    List<JsonObject> accounts = getAccountsForLoan(testLoanId);
+
+	    assertThat(accounts, hasSize(1));
+	    assertThat(getOpenAccounts(accounts), hasSize(0));  
```

Comment:
This indentation is inconsistent with the rest of the file. If that was deliberate, please can you help me understand why. If not, please correct.

Reply:
will correct.